### ISSUE TITLE
Generalization to new data / performance on validation set

### ISSUE THREAD

**Issue by btiv** [btiv/stock-trend-prediction#1]
Training loss is around 0.167, but validation loss around 4.02, need to close this gap and have model do better job generalizing to new data
